### DISCUSSION TITLE
Copy Search Results as CSV to clipboard to paste into Excel

### DISCUSSION THREAD

**Discussion by RezaSeedin** [burkybang/SuiteAdvanced-Feedback#53] [Ideas]
I would like the have the ability to copy search results into my clipboard as CSV to paste into Excel

This saves time downloading and then copy-pasting it again

Let me know what you guys think

**Comment by burkybang** [comment]
Great idea! This'll be in the next version.

**Reply by burkybang** [reply]
Here's a sneak peek
![image](https://user-images.githubusercontent.com/927886/197095781-d2d6e58b-3470-4f8b-8baa-8ccce5f1c9cc.png)
